pennylane/transforms/core/transform_dispatcher.py

```diff
@@ -405,6 +438,7 @@ def __eq__(self, other: object) -> bool:
             and self.transform == other.transform
             and self.kwargs == other.kwargs
             and self.classical_cotransform == other.classical_cotransform
+            # and self.plxpr_transform == other.plxpr_transform
```

Comment:
This doesn't work how we want for equality, and honestly, doesn't really make sense for the purposes of equality anyway. Happy to hear thoughts.

Reply:
Given `self.transfrom` is enought to fully determine the plxpr tranform, classical cotransform, `is_informative`, and `final_transform`, I think we should be good to leave it out.